src/blog/2023/10/blueprints.md

```diff
@@ -0,0 +1,37 @@
+---
+title: What are FlowFuse Blueprints?
+subtitle: Preconfigured Node-RED Applications
+description: Explore the concept of FlowFuse Blueprints, the pre-built solutions to simplify Node-RED experience, and take a look at the first three Blueprints we've launched for manufacturing applications.
+date: 2023-10-16
+authors: ["marian-demme"]
+#image: "/blog/2023/10/images/ANDON1.png"
+tags:
+    - posts
+    - releases
+    - flowfuse
+    - changelog
+---
+
+Starting today, FlowFuse Blueprints are available on FlowFuse Cloud. Additionally, upon request, all our Teams and Enterprise Self-Hosted customers gain access to this collection. But what exactly are FlowFuse Blueprints?
```

Comment:
Follow up to do this week; create docs around blueprints and describe how customers request and upload our blueprints.